provision-contest/ansible/roles/domserver/templates/nginx-domjudge-inner.j2

```diff
@@ -69,5 +69,10 @@ add_header X-Content-Type-Options "nosniff";
 add_header X-XSS-Protection "1; mode=block";
 add_header X-Robots-Tag "none" always;
 
+{% if NGINX_REAL_IP_FROM is defined %}
```

Comment:
This is the name of the solution, not of the feature.

I propose to call it something like `FRONT_REVERSE_PROXY` so people know when to set this.